controllers/books.js

```diff
@@ -1,45 +1,66 @@
 import { v4 as uuidv4 } from 'uuid';
+let books = [{
```

Comment:
You don't have to move the json into the controller. Import the json into the `books` here.